src/hooks/useMoneyRaised.js

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react'
+import moment from 'moment'
+import { isChromaticEnv } from '../utils/featureFlags'
+
+// These must match the new tab page's numbers.
+// Eventually call the API instead of hard-coding.
+const MONEY_RAISED = 1057100.0
+const MONEY_RAISED_UPDATE_TIME = moment('2021-01-07T16:30:00.000Z')
+const DOLLARS_PER_DAY_RATE = 700.0
+
+const useMoneyRaised = () => {
+  const calculateMoneyRaised = () => {
+    const secsInDay = 60 * 60 * 24
+    const now = moment()
+    const diff = now.diff(MONEY_RAISED_UPDATE_TIME, 'seconds')
+    const daysDiff = diff / secsInDay
+    if (isChromaticEnv()) {
+      return MONEY_RAISED
+    }
+    return MONEY_RAISED + daysDiff * DOLLARS_PER_DAY_RATE
+  }
+
+  const initialMoneyRaised = calculateMoneyRaised()
+  const [moneyRaised, setMoneyRaised] = useState(initialMoneyRaised)
```

Comment:
I think this needs to return null until second render because the initial value will be set a build time but then differ at rehydration time, when the React app starts up. React expects the rendered content to exactly match the prerendered content. If they differ, I'd expect the money raised value to display its _build time_ value until the app hydrates—or, worse, there to be some hydration bug that breaks layout. In the current money raised logic, you'll see we wait to render anything until after mount.

For more info, check out:
* https://github.com/gatsbyjs/gatsby/discussions/17914
* [The Perils of Rehydration](https://www.joshwcomeau.com/react/the-perils-of-rehydration/)

Downstream, presentational components using this value would need to handle a null value.